nil/tests/rpc_node/call_test.go

```diff
@@ -133,7 +133,7 @@ func (s *SuiteRpcNodeCall) TestCall() {
 		}
 
 		_, err = s.DefaultClient.EstimateFee(s.Context, callArgs, "latest")
-		s.ErrorContains(err, db.ErrKeyNotFound.Error())
+		s.NoError(err, db.ErrKeyNotFound.Error())
```

Comment:
Could you please describe how changes from this patch (Local readonly APIs are used for all shards on validators and archive nodes) fixed this issue?

Reply:
By calling `api.nodeApi.Call` inside `handleOutTransactions` we stop going off the network, hitting a potentially lagging archive node, and instead process the entire request chain locally. This works well. I need to better deal with what you write above about possible lag of shards that are not validated by the validator, but in fact now we don't encounter problems when the whole chain is processed inside the validator (albeit with calls through the network) in the absence of an archive node.